app/models/concerns/user_edits.rb

```diff
@@ -0,0 +1,45 @@
+module UserEdits
+  class Edits
+    PER_PAGE = 20
+    EDITABLE_TYPES = %w[Relationship Entity List Document].freeze
+    UserEdit = Struct.new(:resource, :version, :action, :time)
```

Comment:
this seems like a really useful struct. (as does the presenter that leverages it.)

perhaps it might be extractable into a common class somewhere that other edits pages could reuse? (as a way to keep yourself from having to reinvent edit formatting every time and also to prevent the format from straying slightly from page to page in a way that confuses @kconn ?)

Reply:
yeah, a refactor down the line would be to standardize all the modification pages